deploy-if-version-bump.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+if ! git rev-parse `npm run version` 2>/dev/null ; then
+    git tag `npm run version`
+    git push --tags  # update the repository version
+    ssh deploy-fr-openfisca-org@fr.openfisca.org
```

Comment:
Why deploying only when a new version has been published ?

Here the context is quite different than publishing a package like OpenFisca Core. You don't have re-users, so the versioning policy doesn't need to be as strict (and actually, so far you don't enforce a version bump for each change).

With this code, you will publish changes **only** if the version has been bumped. It seems like future trouble for the person in 2 months who will wonder why their changes are not deployed in production, while the tests are passing on circle !

Reply:
For rollbacks. It's easier with a versioned content.

Reply:
What rollback can you do with a version that you cannot do with git ?

Reply:
This seems even worse for rollbacks. Let's imagine that:

- A change _A_ is merged in master, without version bump. It is therefore not deployed.
- A change _B_ is merged in master, this time _with_ a version bump. 
- A deployment is triggered, and both _A_ and _B_ are put in production
- A bug appears on the website : which change is responsible ? The developper will probably assume it's B. What if it's A ? What if A has been merged 2 weeks ago ? What if we want to revert A, but we need B to be in prod ?

Reply:
Yes, version bump enforcing is missing in this solution 😅. Nevertheless, I agree that this might be overkill even if we have some kind of reusers (4 or 5 of us might update this website and Tunisia re-used parts of it). 
So, I'm removing the condition on version number and tag and I will ask for consensus to check our needs.